nifi-extension-bundles/nifi-cdc/nifi-cdc-mysql-bundle/nifi-cdc-mysql-processors/src/main/java/org/apache/nifi/cdc/mysql/processors/CaptureChangeMySQL.java

```diff
@@ -1253,7 +1253,9 @@ protected TableInfo loadTableInfo(TableInfoCacheKey key) throws SQLException {
                 for (int i = 1; i <= numCols; i++) {
                     // Use the column label if it exists, otherwise use the column name. We're not doing aliasing here, but it's better practice.
                     String columnLabel = rsmd.getColumnLabel(i);
-                    columnDefinitions.add(new ColumnDefinition(rsmd.getColumnType(i), columnLabel != null ? columnLabel : rsmd.getColumnName(i)));
+                    columnDefinitions.add(new ColumnDefinition(
+                            rsmd.isSigned(i), rsmd.getColumnType(i), columnLabel != null ? columnLabel : rsmd.getColumnName(i)
```

Comment:
It would be helpful to assign these arguments to variables for readability, instead of the inline ternary.